base/Dockerfile

```diff
@@ -19,6 +19,8 @@ RUN apt update -y && \
         wget \
         ca-certificates
 
+COPY untar.sh /usr/local/bin/untar
```

Comment:
`untar` doesn't feel like it covers everything the script is doing. What do you think about something like `lnls-get-install` instead?

Reply:
> `untar` doesn't feel like it covers everything the script is doing.

Indeed it misses the `wget` part. I had considered `wuntar`, but it starts to get weird.

> What do you think about something like `lnls-get-install` instead?

I've avoided using "install" because installing a source code is not a terminology I'm used to. If you think that's okay to call extracting an archive in the local directory as install, we may stick with it.

I dispensed with using "lnls" as it is just an internal utility (although it still resides on the IOC filesystem afterwards). We may add it if you think it helps distinguish the special purpose scripts we create.

Reply:
As we discussed, `lnls-` is interesting due to namespacing.